gradle.properties

```diff
@@ -22,3 +23,7 @@ exo_version=0.1.5+1.19
 
 # https://maven.wispforest.io/io/wispforest/owo-lib/
 owo_version=0.8.2+1.19
+
+litematica_file=3929766
```

Comment:
Please link maven or files pages of all dependencies, otherwise updating them down the line is painful

Reply:
I've added malilib maven, but there isn't for litematica. (Well there is, but it was last updated 2 years ago)

Reply:
I'm aware of this, which is why I specifically mentioned the "Files" page on CurseForge. There should be a link that leads directly to a list of versions for each dependency (be that on a maven repository, CurseForge, Modrinth, whatever), searching them up every time you want to update wastes a lot of time

Reply:
Pushed that

Reply:
Now we're 50% of the way there - when re-reading my initial comment I quite clearly see myself mentioning *all dependencies*

Reply:
Is that right?

Reply:
Ideally you'd link directly to where the Malilib artifacts are stored, but yes - this works well enough